images/docker/assemble-docker.sh

```diff
@@ -7,13 +7,12 @@ set -ex
 
 # Usage: ./assemble-docker.sh Dockerfile.amd64 amd64 3.1.0 typedb ubuntu
 
-DOCKERFILE=$1
+DOCKERFILE=$(realpath $1)
 PLATFORM=$2
-DOCKER_VERSION=$3
+TAG=$(cat $3)
```

Comment:
I've worked it back into bazel, so I sadly need to rely on a file for the tag.
I've removed the assumption that we want VERSION-PLATFORM as tag, and just accept a tag directly.